### ISSUE TITLE
bluguide missing traits

### ISSUE THREAD

**Issue by Cellcubed** [Windower/Lua#1967]
I noticed 3 missing traits in bluguide. The missing traits are Evasion bonus, Resist Silence, and Resist Slow. These are listed in the traits.lua but for some reason not showing up in the UI. I looked at all the code to see if I could fix it but it's too advanced for me. I also noticed voracious trunk has the wrong id in traits.lua. I changed mine from 576 to 579 and voracious trunk works as intended now but still missing the 3 traits.

**Comment by RubenatorX** [comment]
couple of things to note:
1) it will only show traits associated with spells you know, so if you are missing spells then, you may not see all traits
2) depending on the resolution of your screen, some traits will overflow off the right side of the screen as well. Minimizing trait sections can help reveal more

**Comment by Cellcubed** [comment]
Thank you for taking the time to read my issue and respond. 

To your first note: The addon does show me traits that I don't currently know the spells for. It also isn't showing the traits that I listed even though I do have those spells.

To your second note: I noticed this as well and have minimized every trait just to be certain.

Do you see the traits, Evasion bonus, resist silence, and resist slow on your screen while using this addon?

**Comment by RubenatorX** [comment]
> Do you see the traits, Evasion bonus, resist silence, and resist slow on your screen while using this addon?

My blue mage is lvl 36, so no, I only see 7 traits

**Comment by Cellcubed** [comment]
Well my addon shows every trait except the 3 I listed. My blue mage is 99. I have spent many hours using these addons. This is a real bug. The addon also lists the number of spell slots incorrectly at certain levels before 71, due to the author of the addon using subjoblevel for a calculation instead of mainjoblevel.

**Comment by Cellcubed** [comment]
After finding voracious trunk had an incorrect id number, I compared every spell id in traits.lua and spellinfo.lua to Windower/res/spells.lua. I did not find any other discrepancies. I am assuming the ids are correct in windower/res/spells.lua. However, seeing two different blue mage addons not function as intended makes me wonder if the spell ids are all correct. Or perhaps something has changed that has caused addons to not see correctly which spells you have learned. On the other hand, //spbk seems to be working correctly for blue mage so maybe it's just the code for bluguide and BLUAlert

**Comment by Cellcubed** [comment]
I have solved the missing traits problem. How can I add these changes so that other users can download the fixed code? In bluguide.lua, 3 lines must be added, 1 for each trait that doesn't appear. The list of traits starts at line 102. I'm very new to github, I only created this account for issues with windower addons. I have made the appropriate edits and created pull requests for the errors I found with this addon. Sorry if I did anything weird, not sure what a pull request is but i did it in the hopes of this fix getting out to other blue mages.
